ui-test/package-lock.json

```diff
@@ -0,0 +1,1904 @@
+{
```

Comment:
Since we are using `yarn` everywhere else, can you please generate yarn.lock instead?